src/functions.js

```diff
@@ -126,16 +126,27 @@ defineFunction("\\sqrt", {
     };
 });
 
-// Some non-mathy text
-defineFunction("\\text", {
+// Non-mathy text, possibly in a font
+var textFunctionStyles = {
+    "\\text": undefined, "\\textrm": "mathrm", "\\textsf": "mathsf",
+    "\\texttt": "mathtt", "\\textnormal": "mathrm", "\\textbf": "mathbf",
+    "\\textit": "textit", "\\textsc": "textsc",
```

Comment:
Right you are.  My question should've been why are the other `\textXX` defined as aliases to their `\mathXX` counter parts, but `textit` is not?